internal_transfers/actions/src/imbalance.ts

```diff
@@ -0,0 +1,59 @@
+import { TokenImbalance, TransferEvent } from "./models";
+
+export type ImbalanceMap = Map<string, bigint>;
+
+/**
+ * Aggregation operator transforming a list of `TransferEvent` into an
+ * aggregation of token imbalances as an `ImbalanceMap`.
+ * Imbalances are defined in reference to `focalAccount`
+ * @param transfers - a list of token transfers
+ * @param focalAccount - account for which the "imbalance" is referring to.
+ */
+export function aggregateTransfers(
+  transfers: TransferEvent[],
+  focalAccount: string
+): ImbalanceMap {
+  let accumulator: ImbalanceMap = new Map<string, bigint>();
```

Comment:
Late to the party, but here you should use a `const`
You don't reassign the variable later, you only mutate the map it holds.

Also, you could do this using a `reduce` instead, which makes the variable unnecessary.

Something like:

```
return transfers.reduce((acc, transfer) =>{...}, new Map<string, bigint>())
```

Reply:
Oh so fancy! I definitely want to do this. Will turn your comment into an issue.

Reply:
#238